libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/telemetryevents/TelemetryDepartureEvent.kt

```diff
@@ -0,0 +1,20 @@
+package com.mapbox.navigation.core.telemetry.telemetryevents
+
+import android.annotation.SuppressLint
+import android.os.Parcel
+import com.mapbox.android.telemetry.Event
+
+/**
+ * Documentation is here [https://paper.dropbox.com/doc/Navigation-Telemetry-Events-V1--AuUz~~~rEVK7iNB3dQ4_tF97Ag-iid3ZImnt4dsW7Z6zC3Lc]
+ */
+
+// Defaulted values are optional
+
+@SuppressLint("ParcelCreator")
+class TelemetryDepartureEvent(
```

Comment:
Same as above `NavigationDepartEvent`

Reply:
👍

Reply:
TelemetryDepartureEvent has replaced NavigationDepartueEvent. In fact all Navigation<Type>Event have been replaced by Telemetry<Type>Event